### ISSUE TITLE
TT1 Blocks: Latest Comments Block: Migrate CSS to theme.json

### ISSUE THREAD

**Issue by scruffian** [WordPress/theme-experiments#122]
The code is [here](https://github.com/WordPress/twentytwentyone/tree/trunk/assets/sass/05-blocks/latest-comments).

From #82

**Comment by pbking** [comment]
There is a fair bit of block-specific styling that I'm unsure how can/will be supported by Global Styles:

```
.wp-block-latest-comments .wp-block-latest-comments__comment {
.wp-block-latest-comments .wp-block-latest-comments__comment-meta {
.wp-block-latest-comments .wp-block-latest-comments__comment-date {
.wp-block-latest-comments .wp-block-latest-comments__comment-excerpt p {
```
These selectors from the original TT1 theme outlined specific font-sizes.

There's also some first/last child padding constraints as well.

These are all details that MIGHT be configurable and defined in the JSON or it might just be expressed in CSS as specifics to the theme.  I'm unsure what is correct here.

**Comment by kjellr** [comment]
> There is a fair bit of block-specific styling that I'm unsure how can/will be supported by Global Styles:

My general thought here is "wait and see". 😄 

Eventually, I do expect that the comments block will be made up of a series of child blocks, and we'll be able to move and adjust them as we can with things inside of a Query block today. But in the near term, I not sure that it makes a lot of sense to focus on implementing any of these as stopgaps.

**Comment by pbking** [comment]
Indeed, that sounds the best way to be able to be able to express those.

Is there an example of how child-blocks can be styled when within a parent block?  I'm wondering what the theme.json would look like when trying to express these choices as the defaults for the theme.  How does one configure the style of a child block for a particular block?

**Comment by kjellr** [comment]
I don't think there's an example like that yet — I think the closest example would be the Navigation block, where it only accepts a slim range of special innerblocks. But that doesn't include any special innerblock styling options yet.

**Comment by scruffian** [comment]
Would we need to specify how child blocks are styled _when inside_ a child block, or would it be sufficient to simply define defaults for these blocks?

**Comment by kjellr** [comment]
I imagine just defaults for those blocks. But some blocks _only_ live within other blocks (for example, the "Column" block can only live within the "Columns" block), so I'm not sure how those fit into global styles exactly.
